packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -272,6 +272,27 @@ export function getValidatorApi({
     produceBlockV2: produceBlock,
     produceBlindedBlock,
 
+    async getBlob(blockRoot, index) {
+      const blockRootHex = toHex(blockRoot);
+      const blob = (chain.producedBlobSidecarsCache.get(blockRootHex)?.blobSidecars ?? []).slice(index, index + 1)[0];
```

Comment:
Why `.slice(index, index+1)[0]` instead of just `[index]`?

Reply:
thanks for spotting, my head must be spinning